public/components/notebooks/components/__tests__/note_table.test.tsx

```diff
@@ -178,22 +175,21 @@ describe('<NoteTable /> spec', () => {
         dateModified: 'date-modified',
       },
     ];
-    const { getByText, getByLabelText, queryByText } = renderNoteTable({ notebooks });
+    const { getByText, getByLabelText, getAllByText, getByTestId } = renderNoteTable({ notebooks });
 
     // Select a notebook
     fireEvent.click(getByLabelText('Select this row'));
 
-    // Open Actions dropdown and click Delete
-    fireEvent.click(getByText('Actions'));
-    fireEvent.click(getByText('Delete'));
+    // Click the delete button
+    fireEvent.click(getByTestId('deleteSelectedNotebooks'));
 
     // Ensure the modal is open
-    expect(getByText('Delete 1 notebook')).toBeInTheDocument();
+    expect(getAllByText('Delete 1 notebook')).toHaveLength(2);
 
     // Close the delete modal
     fireEvent.click(getByText('Cancel'));
 
     // Ensure the delete modal is closed
-    expect(queryByText('Delete 1 notebook')).toBeNull();
+    expect(getAllByText('Delete 1 notebook')).toHaveLength(1);
```

Comment:
are the comments still accurate? it says ensure modal is closed, but still expects model text to be found?

Reply:
The new functionality adds the delete button to the left of the search bar containing the text. Once the modal is closed the expected behavior is that it is still selected so we still have the (new) button.
<img width="1261" alt="Screenshot 2024-09-03 at 5 10 51 PM" src="https://github.com/user-attachments/assets/692b4dc6-524f-4fa6-97ce-d2ec8e043858">